ticketbot.py

```diff
@@ -23,7 +21,7 @@
 
 
 MatchSet = namedtuple('MatchSet',
-    ['tickets', 'github_changesets', 'github_PRs'])
+                      ['tickets', 'github_changesets', 'github_PRs'])
```

Comment:
Following Django's style, we would prefer a single longer line (and we would also prefer not to make unrelated cosmetic changes in a PR that changes functionality, but whatever).